spec/units/transcoding_spec.rb

```diff
@@ -82,6 +87,28 @@ def generate_special_derivatives
     end
   end
 
+  describe "with an attached RAW image", unless: ENV['TRAVIS'] == 'true' do
```

Comment:
I was emulating some of the other tests in the suite (XLS, TIFF, PPT, etc) that also have this logic. It should run without any special environment settings although I will have to double check if RAW support needs to be compiled into the default gm toolchain or it comes along for free.